libs/components/src/lib/file-picker/file-picker.ts

```diff
@@ -255,10 +241,19 @@ export class FilePicker extends FormAssociatedFilePicker {
 			this.setFormValue(formData);
 		}
 
-		// For historical reasons, the value IDL attribute prefixes the filename with the string "C:\fakepath\".
+		// Like native input[type=file], set the value to a fakepath
 		this.value = files.length > 0 ? `C:\\fakepath\\${files[0].name}` : '';
```

Comment:
Extract the logic to a global module function and use it instead of the comment.
More durable, more self-explanatory code IMO.

```suggestion
		this.value = setFakePathLikeNativeInput(files);
```